### PR TITLE
feat: pass context to push/result function from js-parser

### PR DESCRIPTION
**Pull request by maksimr**

### CODE REVIEW

File: test/parser/parse-jsSpec.js [hunk 1]
```diff
@@ -17,6 +17,33 @@ test('should parse javascript code', (t) => {
 });
 
 
+test('should pass context to the push/result function', (t) => {
+  const result = [];
+  const rules = convertRuels(['foo:0']);
```
Comment (princed) [inline test/parser/parse-jsSpec.js]: Type in helper name

**Review by princed — CHANGES_REQUESTED** [review]

File: test/parser/parse-jsSpec.js (hunk 1 above)
Comment (maksimr) [reply test/parser/parse-jsSpec.js]: done

Comment (princed) [reply test/parser/parse-jsSpec.js]: I meant `convertRuels` function name =)

Comment (maksimr) [reply test/parser/parse-jsSpec.js]: up

**Review by princed — APPROVED** [review]

**Pull request merged**